src/Symfony/Component/Messenger/Worker.php

```diff
@@ -132,7 +133,7 @@ private function handleMessage(Envelope $envelope, ReceiverInterface $receiver,
         ];
 
         try {
-            $envelope = $this->bus->dispatch($envelope->with(new ReceivedStamp($transportName)));
+            $envelope = $this->bus->dispatch($envelope->with(new ReceivedStamp($transportName), new ConsumedByWorkerStamp()));
```

Comment:
Is there anything the worker might want to pass to the chain? E.g. the `$output` of the command? I'm just wondering :)

Reply:
Even though this stamp won't be "sent" on redelivery, it feels odd to pass what's really a "service" into a Stamp. Better might be (if someone needs this) to add it to one of the worker events we dispatch.

Reply:
I see no issues on my side doing so , the day we need to.